azure-pipelines.yml

```diff
@@ -25,6 +25,14 @@ jobs:
       - template: ci/build-unix.yml
         parameters:
           name: linux
+      - bash: |
```

Comment:
hoping azures' yaml parser supports this: can we use [anchors and merge keys](https://camel.readthedocs.io/en/latest/yamlref.html#merge-keys) to reduce code duplication?

Reply:
> /azure-pipelines.yml: Anchors are not currently supported. Remove the anchor 'SLACK'

Nope :(

Reply:
Meh :-/

https://github.com/Microsoft/azure-pipelines-yaml/issues/187